core/src/main/java/org/web3j/protocol/rx/Web3jRx.java

```diff
@@ -11,8 +11,6 @@
 import org.web3j.protocol.core.methods.response.Transaction;
 import org.web3j.protocol.websocket.events.LogNotification;
 import org.web3j.protocol.websocket.events.NewHeadsNotification;
-import org.web3j.protocol.websocket.events.PendingTransactionNotification;
-import org.web3j.protocol.websocket.events.SyncingNotfication;
 
 /**
  * The Observables JSON-RPC client event API.
```

Comment:
Can we change Observables everywhere here to Flowable?

Also, given that we can only rename methods in a major version change I do think we should review the names. How about something like the following:
Instead of `catchupToLatestAndSubscribeToNewBlocksObservable` we have `replayAllAndFutureBlocksFlowable` - you get the gist.